src/constants/gl_markers.js

```diff
@@ -16,6 +16,42 @@ module.exports = {
     'square-open': '□',
     diamond: '◆',
     'diamond-open': '◇',
-    cross: '+',
-    x: '❌'
-};
+    cross: '➕',
```

Comment:
Looks like you changed the `cross` symbol which is breaking the [`gl3d_marker_arrays`](https://github.com/plotly/plotly.js/blob/master/test/image/mocks/gl3d_marker-arrays.json) and the [`gl3d_projection_traces`](https://github.com/plotly/plotly.js/blob/master/test/image/mocks/gl3d_projection-traces.json) mocks.